src/logs.rs

```diff
@@ -35,30 +35,49 @@ pub async fn generate(home: &str) -> anyhow::Result<String> {
     let temp = tempdir.path();
 
     let _ = futures::join!(
-        command("df", &["-h"], temp),
-        command("dmesg", &[], temp),
-        command("dmidecode", &[], temp),
-        command("journalctl", &["--since", "yesterday"], temp),
-        command("lsblk", &["-f"], temp),
-        command("lspci", &["-vv"], temp),
-        command("lsusb", &["-vv"], temp),
-        command("sensors", &[], temp),
-        command("upower", &["-d"], temp),
-        command("uptime", &[], temp),
+        command("df", &["-h"], temp, "df"),
+        command("dmesg", &[], temp, "dmesg"),
+        command("dmidecode", &[], temp, "dmidecode"),
+        command("efibootmgr", &["-v"], temp, "efibootmgr"),
+        command("journalctl", &["--since", "yesterday"], temp, "journalctl"),
+        command(
+            "lsblk",
+            &[
+                "-o",
+                "NAME,MODEL,FSTYPE,FSVER,SIZE,FSUSE%,MOUNTPOINTS,LABEL,UUID"
+            ],
+            temp,
+            "lsblk"
+        ),
+        command("last", &[], temp, "last"),
+        command("lspci", &["-vv"], temp, "lspci"),
+        command("lsusb", &["-vv"], temp, "lsusb"),
+        command("lsmod", &[], temp, "lsmod"),
+        command("sensors", &[], temp, "sensors"),
+        command("systemd-analyze", &["blame"], temp, "systemd-blame"),
```

Comment:
It looks like you added the ability to name files differently than the commands just for this one? Why does this file need to be named `systemd-blame` when there is no such command?

Reply:
Because `systemd-analyze` can output other things as well. This particular log is _specifically_ the output of the command `systemd-analyze blame`, as distinct from (for example) `systemd-analyze dump` or some other sub-command. This addition to the function is only used for this one log _right now_, but it makes it much more clear what that log actually is when you look at the list of files, and it adds flexibility for adding or changing logs in the future.

Reply:
Lots of the other commands are outputting options that aren't part of the filenames. I disagree that this is adding any value and think it's adding unnecessary complexity, but since it passed engineering review, we won't withhold approval over it.

Reply:
I would think that if you're deviating from the command, something like `boot-benchmark` or `startup-benchmark` would be more clear than `systemd-blame`, which someone without any background might not understand at all.

Reply:
We're already renaming files, like `/var/log/apt/term.log.1.gz` is renamed to `apt/term-rotated.log.gz` for clarity, this just adds the same ability to this function that uses output from commands instead. And that's a fair point. I'll go ahead and make the output filename clearer, and look through the other filenames to see if similar changes can be made; I'm thinking that renaming `last` to `reboot-history` would be a worthwhile change too.

Reply:
I would have gone with `disk-usage` for `df -h`, although I can understand that might get confusing since `du` is also a command.